packages/flame/CHANGELOG.md

```diff
@@ -18,9 +18,9 @@
  - Refactor TextBoxComponent
  - Fix bugs with TextBoxComponent
  - Improve error message for composed components
+ - Fix `game.size` to take zoom into consideration (which also fixes `camera.followComponent` when `zoom != 1`)
```

Comment:
I would split this into two entries, one for the fix on game.size and other for the followcomponent fix

Reply:
Done!